seichi-onp-k8s/manifests/seichi-kubernetes/apps/seichi-minecraft/redisbungee-redis.yaml

```
@@ -21,6 +21,11 @@ spec:
             runAsNonRoot: false
           service:
             type: LoadBalancer	
+          resources:
+            requests:
+              cpu: "250m"
+              # https://github.com/GiganticMinecraft/seichi_infra/issues/468
+              memory: "1Gi"
```

これよくわかってないんですけど、これってコンテナレベルのメモリ量制限にはなりますけど、設定しないといけないのって Redis のアプリケーション側の `maxmemory` なはず([参考](https://github.com/oliver006/redis_exporter/issues/300#issuecomment-521733222))で、これが**Helm Chart内で自動で `requests` から引っ張ってこられない限り**(ここが要検証)は

```suggestion
          extraFlags:
            - "--maxmemory 1024mb"
```

になると思います

---

https://github.com/bitnami/charts/search?q=maxmemory
検索した感じそういう設定は特になかったので、フラグで情報渡さないと無理な気がする

---

正しい